### DISCUSSION TITLE
Question about new brightness OSD

### DISCUSSION THREAD

**Discussion by d-anez3** [seerge/g-helper#425] [General]
This isn't really a bug so I'm putting it here instead of issues but is there a reason that when G-Helper starts using its own OSD when Asus Optimization is disabled, that the Windows built-in display brightness OSD is not used like it is when changing volume? It's nice being able to see what level the brightness is at like the Windows one. I suppose I can always look in Control Centre but yeah.

**Comment by seerge** [comment]
@d-anez3 but asus optimization also used own osd for brightness :) I'm not sure if it's possible to access windows native OSD

**Reply by seerge** [reply]
@d-anez3 but what I can do - is to show current brightness level inside OSD like here : 
[GHelper.zip](https://github.com/seerge/g-helper/files/11524995/GHelper.zip)

**Reply by AudreyAP** [reply]
Are you sure? It looked identical to the volume one on my machine...

**Reply by seerge** [reply]
[GHelper.zip](https://github.com/seerge/g-helper/files/11529239/GHelper.zip)
Try this version, it should trigger native windows brightness controlls

**Reply by AudreyAP** [reply]
You've done it again - works perfectly.
